sdk/storage/Azure.Storage.Blobs/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Release History
 
+## 12.3.0 (2020-02)
+- Bumped to service version 2019-07-07.
+- Added support for Encryption Scopes.
+- Modified BlockBlobClient.Upload() and .UploadAsync() to support parallel and multi-part uploads.
+- Add PageBlobClient.GetManagedDiskPageRangesDiff() and .GetManagedDiskPageRangesDiffAsync()
```

Comment:
Should we include this since it's not really targeted at regular customers?

Reply:
Removing line about GetManagedDiskPageRangesDiff().